raiden/ui/cli.py

```diff
@@ -803,35 +808,32 @@ def prompt_account(address_hex, keystore_path, password_file):
         )
         sys.exit(1)
 
-    if not accmgr.address_in_keystore(address_hex):
-        # check if an address has been passed
-        if address_hex is not None:
-            print("Account '{}' could not be found on the system. Aborting ...".format(
-                address_hex))
-            sys.exit(1)
-
-        addresses = list(accmgr.accounts.keys())
+    if not address_hex:
         formatted_addresses = [
-            '[{:3d}] - {}'.format(idx, to_checksum_address(addr))
-            for idx, addr in enumerate(addresses)
+            to_checksum_address(addr)
+            for addr in accmgr.accounts.keys()
```

Comment:
While you're at it, could you add a `sorted()` call here? Having the addresses in random order has always been annoying.